types/oracledb/index.d.ts

```diff
@@ -1308,7 +1307,7 @@ declare namespace OracleDB {
             operation: number;
             /** ROWID of the row that was affected. */
             rowid: string;
-        }[] | undefined;
+        }[];
```

Comment:
Having an explicit `| undefined` on optional properties is useful for TypeScript’s `exactOptionalPropertyTypes`: <https://www.typescriptlang.org/tsconfig#exactOptionalPropertyTypes>

```suggestion
        }[] | undefined;
```